README.md

```diff
@@ -1,44 +1,251 @@
 ![zero](https://github.com/commitdev/zero/blob/master/docs/img/logo-small.png?raw=true)
 
-This is a WIP version of our tool Zero which aims to make it quick and easy for developers to bootstrap a new environment and get to writing code.
+## What is Zero
 
-For the older, simple tool that just rendered templates, see [commitdev/zero-old][old]
+Zero is an open-source developer platform CLI tool which makes it quick and easy for technical founders & developers to build quality and reliable infrastructure to launch, grow and scale high-quality SaaS applications faster and more cost-effectively.
 
-## Press Release
-Learn more about Zero's vision here:
-https://docs.google.com/document/d/1YNRNgCfCHCxmIpD5ZsLYG2xCBxJLFd6CBI0DS_NFqoY/edit
+## Why is zero good for startups
 
-## Project Board
-https://app.zenhub.com/workspaces/commit-zero-5da8decc7046a60001c6db44/board?filterLogic=any&repos=203630543,247773730,257676371,258369081
+As a technical founder or the first technical hire at a startup, your sole focus is to build the logic for your application, and get it into customers’ hands as quickly and reliably as possible. Yet you immediately face multiple hurdles before even writing the first line of code. You’re forced to make many tech trade offs, leading to decision fatigue. You waste countless hours building boilerplate SaaS features not adding direct value to your customers. You spend precious time picking up unfamiliar tech, make wrong choices that result in costly refactoring or rebuilding in the future, and are unaware of tools and best practices that would speed up your product iteration.
+
+## Why is zero Reliable, Scalable, Performant and Secure
+
+Zero leverages Amazons’ Elastic Kubernetes Service. EKS is amazon managed Kubernetes service where you can build and deploy your applications/containers. Zero & EKS is deeply integrated with other AWS services such as:
```

Comment:
```suggestion
Zero's default EKS stack leverages Amazons’ Elastic Kubernetes Service. EKS is Amazon's managed Kubernetes service where you can build and deploy your applications/containers. Zero-AWS-EKS-stack uses a suite of technologies such as:
- [Amazon Virtual Private Cloud][vpc].
- [AWS Identity and Access Management][iam]
- [Amazon Cloud Watch][acw].
- [Auto Scaling Groups][asg].
```

This doesn't really talk about Reliable, Scalable, Performant and Secure

Maybe talk about 

- Reliability: Our infrastructure is built on multiple availability zones making our system highly available and fault tolerant
- Scability: Our EKS infrastructure is built on top of EC2 instances, and set up with AWS's [Auto Scaling Groups][asg] is ready to scale whenever the need arises. 
- Performant: Using Kubernetes leveraging containers for your services reduces overhead on virtual machines
- Security: Properly configured access-control to resources/security groups, using secure vault systems(AWS's secret manager), and following best practices provides great security out of the box.

Reply:
👍🏽